Utilities/Mergers/include/Mergers/ObjectStore.h

```diff
@@ -29,17 +31,27 @@ namespace o2::mergers
 class MergeInterface;
 
 using TObjectPtr = std::shared_ptr<TObject>;
+using VectorOfTObject = std::vector<TObject*>;
+using VectorOfTObjectPtr = std::vector<TObjectPtr>;
 using MergeInterfacePtr = std::shared_ptr<MergeInterface>;
-using ObjectStore = std::variant<std::monostate, TObjectPtr, MergeInterfacePtr>;
+using ObjectStore = std::variant<std::monostate, TObjectPtr, VectorOfTObjectPtr, MergeInterfacePtr>;
 
 namespace object_store_helpers
 {
 
 /// \brief Takes a DataRef, deserializes it (if type is supported) and puts into an ObjectStore
 ObjectStore extractObjectFrom(const framework::DataRef& ref);
 
+/// \brief Helper function that converts vector of smart pointers to the vector of raw pointers that is serializable
+///        Destroy original vector after destroying vector with raw pointers to avoid undefined behavior
+VectorOfTObject toRawPointers(const VectorOfTObjectPtr&);
```

Comment:
```suggestion
VectorOfTObject toRawObserverPointers(const VectorOfTObjectPtr&);
```
perhaps change the name as such to be clearer about the fact that the returned vector does not own the pointers.